src/datachain/lib/udf.py

```diff
@@ -35,42 +37,47 @@ class UDFAdapter(_UDFBase):
     def __init__(
         self,
         inner: "UDFBase",
-        properties: UDFProperties,
+        properties: "UDFProperties",
```

Comment:
minor: seems this change is not needed?

Reply:
or you should move UDFProperties into the `TYPE_CHECKING` section? (overall, same ask - can we do less minor updates or split them if they are not essential to the PR, thanks!)

Reply:
Minor update, leftover, I can revert this change 👍